backend/src/packages/orders/order.controller.ts

```diff
@@ -293,6 +296,22 @@ class OrderController extends Controller {
         ),
     });
 
+    this.addRoute({
+      path: OrdersApiPath.USER,
+      method: 'GET',
+      authStrategy: AuthStrategy.INJECT_USER,
```

Comment:
```suggestion
      authStrategy: AuthStrategy.VERIFY_JWT,
```
if it only for registered user